core/org.wso2.carbon.utils/src/main/java/org/wso2/carbon/utils/DiagnosticLog.java

```diff
@@ -43,7 +43,7 @@ public class DiagnosticLog {
     private final String componentId;
```

Comment:
I see we have removed all the getters. Once the DiagnosticLog object is created, don't we have requirements to access logId etc.?
Basically, how would we consume the data captured in DiagnosticLog object?

Reply:
The deprecation has been removed with b8ad1861746e314a045f80f9ec58ae933043e615